### ISSUE TITLE
feature: View source

### ISSUE THREAD

**Issue by toqueteos** [elm-lang/elm-reactor#196]
**TLDR:** It would be great to have some sort of endpoint and/or option like `/_source` (similar to `/_compile`) to show `.elm` sources. We already have nice syntax coloring (I discovered it by clicking on a `.go` file)

---

I tried writing this on https://github.com/elm-lang/elm-reactor/issues/193 but it's locked to collaborators only.

# Context

I've been trying out Elm recently and when I'm stuck with an error I can't manage to solve by myself I use `elm reactor` + `ngrok http 8000` to let my friends see the bug in its live environment.

Why? It may seem like overkill but it's due to **compiler errors not being copy-paste friendly**.

# Problem

Now, `.elm` files get automatically compiled and shown to the user but sometimes I just want to see their code.

**Comment by evancz** [comment]
Seems reasonable. I have a UI in mind that would permit this pretty nicely. Will add to #193!
